### PR TITLE
Fixing/adding python27 derivations

### PR DESCRIPTION
**Pull request by jpgu-epam**
###### Motivation for this change

As the channel keeps supporting python27 some derivations need to be fixed or added to still working is this python version.

###### Things done

- Add python27 support for the following derivations:    

    GitPython
    Venusian
    Uritools
    Sphinxcontrib-blockdiag

- Adding Protobuf 3.3.0 derivation

- Fix tremor repository which was moved to GitLab

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [x] other Linux distributions
- [x] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -7038,3 +7048,4 @@ in {
 });
 
 in fix' (extends overrides packages)
```
Comment (FRidh) [inline pkgs/top-level/python-packages.nix:7050]: please undo this change

**Review by FRidh — CHANGES_REQUESTED** [review]
Thanks. One commit per change though. In this case that would mean one commit per package.

**Comment by jonringer** [comment]
When backporting changes, please follow https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md#backporting-changes.

Namely, you should be doing `git cherry-pick -x <rev>` from a commit that has already landed in master. If the branches have diverged, you may alter the commit or add another commit to ensure that the package is able to still evaluate and build

**Comment by SuperSandro2000** [comment]
Due to the number of PRs nixpkgs gets every month we didn't review and merge this on in time before 20.03 hit EOL. If you think this PR is still relevant please rebase it on master or 20.09 and leave a comment.
